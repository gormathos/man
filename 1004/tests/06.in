4
9.8012 3.77572 9.65574 8.68423
6
5.23688 6.16982 8.65806 4.9107 1.9484 3.95565
