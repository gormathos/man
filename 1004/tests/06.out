1.00000 -31.91689 369.85956 -1817.16064 3103.10100
-101.59433 -76.20220 -0.14572 -99.38690 744.96313 -28.34924
5.18980 41.32139 5.81194 -19.96131 -709.80005 -141.73857
